src/Compilers/CSharp/Portable/Lowering/Instrumentation/Instrumenter.cs

```diff
@@ -228,6 +229,19 @@ public virtual BoundExpression InstrumentCall(BoundCall original, BoundExpressio
             return rewritten;
         }
 
+        /// <summary>
+        /// Similarly to an interceptor, gives the instrumenter an opportunity to adjust call target, receiver and arguments.
+        /// </summary>
+        public virtual void InterceptCallAndAdjustArguments(
+            ref MethodSymbol method,
+            ref BoundExpression? receiver,
+            ref ImmutableArray<BoundExpression> arguments,
+            ref ImmutableArray<RefKind> argumentRefKindsOpt,
+            bool invokedAsExtensionMethod,
```

Comment:
>bool invokedAsExtensionMethod [](http://example.com/codeflow?start=12&length=29)

Is this really a thing post lowering?

Reply:
I used the same signature as `LocalRewriter.InterceptCallAndAdjustArguments`